src/Corax/Constants.cs

```diff
@@ -73,6 +73,8 @@ public static class IndexedField
         
         public static class IndexWriter
         {
+            public const long CancellationTokenCheckInterval = (1L << 12) - 1L; 
```

Comment:
What is that about? Can we get a normal value here? Why 4095 and why is it represented like that?